lightning/src/ln/channelmanager.rs

```diff
@@ -2321,8 +2345,11 @@ impl<M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelManager<M, T, K, F
 					let forwarding_id_opt = match id_option {
 						None => { // unknown_next_peer
 							// Note that this is likely a timing oracle for detecting whether an scid is a
-							// phantom.
-							if fake_scid::is_valid_phantom(&self.fake_scid_rand_bytes, *short_channel_id, &self.genesis_hash) {
+							// phantom or an intercept.
+							if (self.default_configuration.accept_intercept_htlcs &&
```

Comment:
Hmmm as we allow the user to generate `Intercept scids` without `config::accept_intercept_htlcs` set to true, would it make sense to add some `logging` here if we receive a valid `Intercept scid` while having `config::accept_intercept_htlcs` set to false, to help the user figure out what they're doing wrong (ie. telling the user to set `config::accept_intercept_htlcs` to true)?

Reply:
I think it would be a bit ugly to add this log. I'd slightly prefer to pass, I do think the config option is mentioned in a ton of places

Reply:
Ok fair enough :)